src/Api/Response/Response.php

```diff
@@ -172,6 +163,54 @@ public function getQueryInfoMessage(): string
         }
     }
 
+    public function getSmartDidYouMean(): string
+    {
+        $dataQueryInfoMessage = $this->getData(self::DATA_QUERY_INFO_MESSAGE);
+
+        if (empty($dataQueryInfoMessage)) {
+            return '';
+        }
+
+        $type = $dataQueryInfoMessage['queryStringType'];
+        $alternativeQuery = $dataQueryInfoMessage['currentQuery'];
```

Comment:
Why isn't the key called `alternativeQuery`? Isn't this only the alternative query when SDYM is triggered and during normal search the query?

Reply:
For searches where the term is improved or corrected, the current query is the one that is found in the response. This would be the alternative query. For the other case it would be did-you-mean.

The key is not called `alternativeQuery`, because we only know that this is either the query that the user sent, or a query that is improved by SDYM.

Reply:
Ok